### DISCUSSION TITLE
No longer usable

### DISCUSSION THREAD

**Discussion by gimplyworxs** [aclap-dev/video-downloadhelper#557] [Help and Q&A]
### Your question

This new version is now totally unusable for me.
Can't identify this super small font anymore (I'm 71).
Looks very modern now, but unusable. Have uninstalled it now.
Too bad.

### Addon & CoApp version and details

xxx

**Comment by paulrouget** [comment]
You're not the first one to complain about the tiny text. I'm gonna add an option to make it bigger in an upcoming version.

**Reply by paulrouget** [reply]
At the moment, there's a way to make the popup bigger (in the settings, click on "make the addon popup larger") but the font are still small, but the window is bigger.

In version v9.0.1.20, this option will also make the font bigger! Hope this will make you happy :)

**Reply by GrampaWildWilly** [reply]
You may be talking to an empty room here, Paul.  The guy said he has already uninstalled VDH.  Plus, in order to get these improvements, he would have to install the beta.  I'm guessing he's not a beta kind of guy.
